src/refmvs.rs

```diff
@@ -34,7 +34,9 @@ use zerocopy::FromZeroes;
 #[derive(Clone, Copy, Default, PartialEq, Eq)]
 #[repr(C, packed)]
 pub struct RefMvsTemporalBlock {
-    pub mv: Mv,
+    /// This is an [`Mv`]. However, because the type [`Mv`] is `align(4)`,
+    /// we cannot include it in this packed type. Include the underlying array instead.
```

Comment:
```suggestion
    /// This is a [`Mv`]. However, because [`Mv`] is `align(4)`,
    /// we cannot include it in this packed `struct`.
    /// So we use the underlying array instead.
```